make/boards/tsb2.board

```diff
@@ -0,0 +1,67 @@
+# Define a board.
+#
+# A board describes a PCB with a microcontroller and peripherals (flash,
+# LEDs, sensors etc) which are used by this firmware build.
+
+# This is the TestSystemBoard2 tsb2
+
+# Determine board name and location (must be done before including any other files)
+BOARD_NAME          := $(notdir $(basename $(lastword $(MAKEFILE_LIST))))
+BOARD_MAKE_DIR      := $(patsubst %/,%,$(dir $(lastword $(MAKEFILE_LIST))))
+
+# Include the architecture definition
+$(call includeArch,efr32xg2x)
+
+MCU_PART            := EFR32BG21A020F1024IM32
+MCU_ARCH            := efr32xg2x
+MCU_FAMILY          := efr32xg21
+
+# Board options for compiler and linker, libraries
+CFLAGS      += -D$(MCU_PART)
+LDFLAGS     += -Wl,-T,$(SILABS_SDKDIR)/platform/Device/SiliconLabs/EFR32BG21/Source/GCC/efr32bg21.ld
+LDLIBS      += $(SILABS_SDKDIR)/platform/radio/rail_lib/autogen/librail_release/librail_efr32xg21_gcc_release.a
+
+INCLUDES    +=  \
+    -I$(SILABS_SDKDIR)/platform/radio/rail_lib/chip/efr32/efr32xg2x  \
+    -I$(SILABS_SDKDIR)/platform/Device/SiliconLabs/EFR32BG21/Include \
+    -I$(SILABS_SDKDIR)/platform/CMSIS/Include
+
+SOURCES     += \
+    $(SILABS_SDKDIR)/platform/Device/SiliconLabs/EFR32BG21/Source/GCC/startup_efr32bg21.c \
+    $(SILABS_SDKDIR)/platform/Device/SiliconLabs/EFR32BG21/Source/system_efr32bg21.c
+
+# Pull in the board support code
+BOARD_SRC_DIR       := $(BOARD_MAKE_DIR)/../../src/boards/$(BOARD_NAME)
+INCLUDES            += -I$(BOARD_SRC_DIR)
+SOURCES             += $(BOARD_SRC_DIR)/platform.c
+
+# Pull in the platform_io.c
+ifndef NODE_PLATFORM_DIR
+$(error NODE_PLATFORM_DIR has not been defined!)
+endif
+SOURCES             += $(NODE_PLATFORM_DIR)/silabs/platform_io.c
+
+# Pull in default OS configuration, after possible platform-specific overrides
+ifneq ($(CMSIS_CONFIG_DIR),)
+INCLUDES            += -I$(CMSIS_CONFIG_DIR)
+endif
+
+# Allocate UUID for this board
+# NB! UNIQUE: MUST NOT BE USED IN OTHER BOARDS.
+# NB! CONST: CANNOT BE CHANGED FOR THIS BOARD.
+UUID_BOARD          := c359a127-46a0-47f2-b827-2c21e284bd59
+
+# Maximum size limits for binaries in memory, their start addresses
+BOOTLOADER_START    := 0x00000
+BOOTLOADER_MAX_LEN  := 0x14000 # 64K+32K
+STAGE1_START        := 0x00000
+STAGE1_MAX_LEN      := 0x10000 # 64K
+STAGE2_START        := 0x10000
+STAGE2_MAX_LEN      := 0x08000 # 32K
+APP_START           ?= 0x18000 #
```

Comment:
Shall we have APP_START as 0 for tsb platform?

Reply:
Not necessarily, will want bootloader in some scenarios also.